Sources/MountebankSwift/Models/Stub/Response/Is/Body.swift

```diff
@@ -34,3 +34,24 @@ public enum Body: Equatable {
         }
     }
 }
+
+extension Body: Recreatable {
+    var recreatable: String {
+        switch self {
+        case .text(let string):
+            return enumSwiftString([string])
+        case .json(let json):
+            return enumSwiftString([json])
+        case .jsonEncodable(let encodable, let encoder):
+            do {
+                let data = try (encoder ?? jsonEncoder).encode(encodable)
+                let json = try jsonDecoder.decode(JSON.self, from: data)
+                return Body.json("").enumSwiftString([json])
```

Comment:
We first need to convert this to JSON to make this `Recreatable`

The `Body.json("")` part is a bit weird, but `enumSwiftString` uses `self` to make a string, and we need the output here to be `.json(...)`.